File: Java/Estrutura Sequencial/Duracao.java
import java.util.Locale;
import java.util.Scanner;

public class Duracao {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int duracao, hora, minutos, segundos;

        System.out.print("Digite a duracao em segundos: ");
        duracao = sc.nextInt();

        hora = duracao / 3600;
        minutos = (duracao % 3600) / 60;
        segundos = (duracao % 3600) % 60;

        System.out.println(hora + ":" + minutos + ":" + segundos);

        sc.close();

    }
}

File: Python/Estrutura Sequencial/Duracao.py
duracao: int; minutos: float; segundos: float; horas: float

duracao = int(input("Digite a duracao em segundos: "))

horas = duracao // 3600
minutos = (duracao % 3600) // 60
segundos = (duracao % 3600) % 60

print(f"{horas}:{minutos}:{segundos}")